oggm/core/flowline.py

```diff
@@ -797,9 +808,10 @@ def __init__(self, flowlines, mb_model=None, y0=0., glen_a=None,
             a = np.zeros(nx+1)
             b = np.zeros(nx+1)
             c = np.zeros(nx+1)
-            d = np.zeros(nx-1)
-            e = np.zeros(nx)
-            self._stags.append((a, b, c, d, e))
+            d = np.ones(nx+1)  # shape factor default is 1
```

Comment:
Set to `1.0` as a scalar if scalar*array is faster than array*array in numpy. Would be cool, if numpy could skip multiplication by 1 automatically